LSE-DSI/lse-ds202-course-notes
# add roadmap and solutions for w04 lab

Dear all,

The roadmap and solution for exercises of w04 lab have been added. Please have a look and let me know if anything needed to be corrected. Thanks!

## Issues

Closes #52 

## How to validate

(edited by @jonjoncardoso)

1. Switch to the branch of this Pull Request:

```console
git fetch origin
git checkout -b issue52-mangoJ0106-lab-w04 origin/issue52-mangoJ0106-lab-w04
```

2. Ensure you have Quarto installed either on RStudio or VSCode (preferred). Instructions for VSCode can be found [here on our README](https://github.com/LSE-DSI/lse-ds202-course-notes#how-to-build-the-course-website).

3. Go to `weeks/week04/lab.qmd` and render the HTML

4. Replicate the lab while thinking about how students in your lab might take those. ((link to relevant slides))[https://lse-dsi.github.io/lse-ds202-course-notes/weeks/week03/lecture.html].

5. If you agree to everything, just come back here to Github, Add your Review and say **Approve**.

6. If you want to ask something to be changed, or make suggestions, come back here to Github, click on Add your Review, select "Request changes". In this case, @mangoJ0106 will make the amendments, commit & push, and request your review again.

7. Once all three of you have accepted this PR, I will merge it to the main website
